Tests/Fixtures/Generated/Storyboards-iOS/swift2-context-all-ignore-module.swift

```diff
@@ -62,43 +64,50 @@ internal enum StoryboardScene {
   internal enum Anonymous: StoryboardType {
     internal static let storyboardName = "Anonymous"
 
-    internal static let initialScene = InitialSceneType<UINavigationController>(Anonymous.self)
+    internal static let initialScene = InitialSceneType<UIKit.UINavigationController>(Anonymous.self)
   }
   internal enum Dependency: StoryboardType {
     internal static let storyboardName = "Dependency"
 
-    internal static let Dependent = SceneType<UIViewController>(Dependency.self, identifier: "Dependent")
+    internal static let Dependent = SceneType<UIKit.UIViewController>(Dependency.self, identifier: "Dependent")
   }
   internal enum Message: StoryboardType {
     internal static let storyboardName = "Message"
 
-    internal static let initialScene = InitialSceneType<UIViewController>(Message.self)
+    internal static let initialScene = InitialSceneType<UIKit.UIViewController>(Message.self)
 
-    internal static let Composer = SceneType<UIViewController>(Message.self, identifier: "Composer")
+    internal static let Composer = SceneType<UIKit.UIViewController>(Message.self, identifier: "Composer")
 
-    internal static let MessagesList = SceneType<UITableViewController>(Message.self, identifier: "MessagesList")
+    internal static let MessagesList = SceneType<UIKit.UITableViewController>(Message.self, identifier: "MessagesList")
 
-    internal static let NavCtrl = SceneType<UINavigationController>(Message.self, identifier: "NavCtrl")
+    internal static let NavCtrl = SceneType<UIKit.UINavigationController>(Message.self, identifier: "NavCtrl")
 
     internal static let URLChooser = SceneType<XXPickerViewController>(Message.self, identifier: "URLChooser")
   }
+  internal enum NoPrefix: StoryboardType {
+    internal static let storyboardName = "NoPrefix"
+
+    internal static let Item1 = SceneType<GLKit.GLKViewController>(NoPrefix.self, identifier: "item 1")
+
+    internal static let Item2 = SceneType<AVKit.AVPlayerViewController>(NoPrefix.self, identifier: "item 2")
+  }
   internal enum Placeholder: StoryboardType {
     internal static let storyboardName = "Placeholder"
 
-    internal static let Navigation = SceneType<UINavigationController>(Placeholder.self, identifier: "Navigation")
+    internal static let Navigation = SceneType<UIKit.UINavigationController>(Placeholder.self, identifier: "Navigation")
   }
   internal enum Wizard: StoryboardType {
     internal static let storyboardName = "Wizard"
 
     internal static let initialScene = InitialSceneType<CreateAccViewController>(Wizard.self)
 
-    internal static let AcceptCGU = SceneType<UIViewController>(Wizard.self, identifier: "Accept-CGU")
+    internal static let AcceptCGU = SceneType<UIKit.UIViewController>(Wizard.self, identifier: "Accept-CGU")
 
```

Comment:
Side note: We should fix that franglicism in the examples one day 😂 (CGU 🇫🇷 = "Conditions Générales d'Utilisation" = TOS 🇺🇸 = "Terms of Service")

Reply:
Never even realised that's what it stood for 😆